translations/Makefile

```diff
@@ -11,11 +11,12 @@ Makefile.coq: _CoqProject
 clean: Makefile.coq
 	$(MAKE) -f Makefile.coq clean
 
-install: all
-	$(MAKE) -f Makefile.coq install
-
-uninstall: all
-	$(MAKE) -f Makefile.coq uninstall
-
 mrproper:
 	rm -f Makefile.coq
+
+force Makefile _CoqProject.in metacoq-config: ;
+
```

Comment:
Just a question, what does force do?

Reply:
The idea is that we want all tasks delegated to `Makefile.coq` to be `.PHONY` (rerun every time). Since `force` is declared `.PHONY` and the delegated task depends on `force`, it becomes `.PHONY`. If there is indeed a file for the delegated task, this will be handled by `Makefile.coq`.